typescript/packages/subsurface-viewer/src/components/Map.tsx

```diff
@@ -1299,9 +1299,9 @@ function getViewState3D(
 
     let width = xMax - xMin;
     let height = yMax - yMin;
-    if (deck) {
-        width = deck.width;
-        height = deck.height;
+    if (deck?.width && deck?.height) {
+        width = deck.width || 1;
```

Comment:
revert

Reply:
Need to do like this since deck may be undefined
    if (deck && deck?.width > 0 && deck?.height > 0) {
        width = deck.width;
        height = deck.height;
    }

If deck is defined then decl.width must be defined (I think)